nh_ews/notifications.py

```diff
@@ -56,7 +56,7 @@ class nh_clinical_notification_medical_team(orm.Model):
     _name = 'nh.clinical.notification.medical_team'
     _inherit = ['nh.clinical.notification']
     _description = 'Inform Medical Team?'
-    _notifications = [{'model': 'doctor_assessment', 'groups': ['nurse']}]
+    _notifications = [{'model': 'doctor_assessment', 'groups': ['doctor']}]
```

Comment:
Might be getting confused here, but the 'Inform Medical Team' task is supposed to be seen (and completed) by the 'Nurse' group, not the 'Doctor' group. Why are we changing this one?